test/FSharp.Azure.Storage.Tests/Table/DataQueryTests.fs

```diff
@@ -405,6 +422,20 @@ let tests connectionString =
                 |> fst
 
             retrievedGame |> Expect.equal "Retrieved game should be correct" game
+            
+            
+        gameTestCase "querying for a record that has option type fields with none value works when filtering by the option-types properties" <| fun ts ->
```

Comment:
Not sure about this test name... seems like it's testing the opposite of "works", in that it's testing that it _doesn't_ work. :)

Perhaps "querying for a record and filtering using a None option value is not supported by table storage"?